requirements.txt

```diff
@@ -4,4 +4,6 @@ PyYAML
 packaging
 rich
 urllib3<3
-linode-metadata
+
+# unpin from git when newer metadata client released
+linode-metadata @ git+https://github.com/linode/py-metadata@main
```

Comment:
We will need a linode_metadata release and remove the pin to the git branch here before merging this PR